cli/mender-artifact/log.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2018 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Log is a global reference to our logger.
+var Log *logrus.Logger
```

Comment:
Do we want to use `mendersoftware/log` instead? It never turned out to be the great logger we thought of, so I'm alright if we don't. But the client uses it and it gives the file that the message came from. It's interface compatible with logrus.